backend/src/test/java/com/darass/darass/DarassApplicationTests.java

```diff
@@ -1,13 +1,12 @@
 package com.darass.darass;
 
-import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 class DarassApplicationTests {
```

Comment:
```suggestion
@SpringBootTest
@ActiveProfiles("test")
class DarassApplicationTests {
```
H2 DB를 연결해 주어야 할 것 같네요.

Reply:
넵!! 반영 했습니다 😃